diff.py

```diff
@@ -200,6 +200,11 @@ def complete_symbol(
         help="Automatically update when source/object files change. "
         "Recommended in combination with -m.",
     )
+    parser.add_argument(
+        "--no-make",
```

Comment:
`dest="no_make"` to make this greppable. Alternatively use `dest="make" action="store_const" const=False` (and const=True for --make) and check for whether make is True, False or None

Reply:
I'll go with `dest="no_make"`, sounds less spaghetti :p